modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java

```diff
@@ -107,6 +107,9 @@ class KafkaToIgniteCdcStreamerApplier implements Runnable, AutoCloseable {
     /** The maximum time to complete Kafka related requests, in milliseconds. */
     private final long kafkaReqTimeout;
 
+    /** Consumer poll timeout. */
```

Comment:
Let's mention in javadocs (especially in public config) that the timeout is measured in "milliseconds", in the same way it is for `kafkaReqTimeout`.